lua/methods/coalitions.lua

```diff
@@ -21,13 +21,158 @@ local skill = {
   Player = 5
 }
 
+-- types for aircraft taking off
+local takeoffTypes = {
+  [1] = "TakeOff",
+  [2] = "TakeOffParking",
+  [3] = "TakeOffParkingHot",
+  [4] = "Turning Point",
+  runway = "TakeOff",
+  cold = "TakeOffParking",
+  hot = "TakeOffParkingHot",
+  air = "Turning Point"
+}
+
+-- actiions for aircraft taking off
+local takeoffActions = {
+  [1] = "From Runway",
+  [2] = "From Parking Area",
+  [3] = "From Parking Area Hot",
+  [4] = "Turning Point",
+  runway = "From Runway",
+  cold = "From Parking Area",
+  hot = "From Parking Area Hot",
+  air = "Turning Point",
+}
+
 --local altitudeType = {
 --  [1] = "BARO",
 --  [2] = "RADIO",
 --  BARO = 1,
 --  RADIO = 2
 --}
 
+local buildAirplanePoints = function(points)
+  local builtPoints = {}
+  for _, pointData in pairs(points) do
+    local pointVec3
+    if type(pointData.place) == "string" then
+      if Airbase.getByName(pointData.place) then
+        pointVec3 = Airbase.getByName(pointData.place):getPoint()
+        env.info("ab point type")
+      elseif trigger.misc.getZone(pointData.place) then
+        pointVec3 = trigger.misc.getZone(pointData.place).point
+        env.info("zone point type")
+      end
+    elseif type(pointData.place) == "table" then
+      pointVec3 = coord.LLtoLO(pointData.place.lat, pointData.place.lon)
+    end
+    builtPoints[#builtPoints+1] = {
+      alt = pointData.alt,
+      x = pointVec3.x,
+      y = pointVec3.z,
+      type = pointData.type or takeoffTypes.air,
+      eta = 0,
+      eta_locked = true,
+      alt_type = pointData.alt_type or "BARO",
```

Comment:
Can you switch this to using https://github.com/DCS-gRPC/rust-server/blob/main/lua/methods/coalitions.lua#L24 (and uncomment it, I only commeneted it because it was not being used yet and making the lint checker complain)